lib/JITIDL/ChakraJIT.idl

```diff
@@ -3,8 +3,13 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
+cpp_quote("#ifndef __JITTypes_h__")
+cpp_quote("#define __JITTypes_h__")
```

Comment:
A bit weird for this header to check that symbol and depend on definition in the other header. Likely able to do the same thing by only touching the other header ("JITTypes.h").